packages/account/src/Sections/Security/ConnectedApps/data-table-template.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { toMoment } from '@deriv/shared';
+import { Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const getConnectedAppsColumnsTemplate = (handleToggleModal) => [
+    { title: localize('Name'), col_index: 'name' },
+    {
+        title: localize('Permission'),
+        col_index: 'scopes',
+        renderCellContent: ({ cell_value }) => {
+            return prepareConnectedAppsScopes(cell_value);
+        },
+    },
+    {
+        title: localize('Last login'),
+        col_index: 'last_used',
+        renderCellContent: ({ cell_value }) => prepareConnectedAppsLastLogin(cell_value),
+    },
+    {
+        title: localize('Action'),
+        col_index: 'app_id',
+        renderCellContent: ({ cell_value }) => prepareConnectedAppsAction(cell_value, handleToggleModal),
+    },
+];
+
+const prepareConnectedAppsAction = (app_id, handleToggleModal) => {
+    return (
+        <Button className='revoke_access' small secondary onClick={() => handleToggleModal(app_id)}>
+            {localize('Revoke access')}
+        </Button>
+    );
+};
+
+const prepareConnectedAppsLastLogin = (last_used) => (
+    <p className='last_used_content'>{toMoment(last_used).format('YYYY-MM-DD HH:mm:ss')}</p>
+);
+
+const oauth_apps_list_map = {
+    read: localize('Read'),
+    trade: localize('Trade'),
+    trading_information: localize('Trading information'),
+    payments: localize('Payments'),
+    admin: localize('Admin'),
+};
```

Comment:
Please use a function to be able to re-evaluate string values for localisation upon language change.

Reply:
done

Reply:
@arash-fs I meant you should make the constant a function. Right now with the change you've made, the strings are just not localised anymore.

```javascript
const oauth_apps_list_map = () => ({
    read: localize('Read'),
    trade: localize('Trade'),
    trading_information: localize('Trading information'),
    payments: localize('Payments'),
    admin: localize('Admin'),
});```

Reply:
aaaa, now make sense. :+1: